### PR TITLE
Param: Color - Theme icon look off centered and color issue when theme color is choose.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/2af43800-6ae0-4fad-a388-d9dd720bd3b0?redirect=true&view=0ce18087-e6b2-4a91-b54c-c386ce1f811c&t=attributes
- `mix-blend-mode` CSS causing issue to make color orange when Theme blue color is selected.

### Screenshots

- https://d.pr/v/SGcFyI

### Types of changes

- Improvement

### How has this been tested?

- See updated ss here - https://share.getcloudapp.com/OAu4kZkG

### Checklist:
- [x] My code is tested
- [x] I've included any necessary tests 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: src/components/color-control/advanced-pop-color-control.js [hunk 1]
```diff
@@ -142,7 +142,26 @@ const AdvancedPopColorControl = ( props ) => {
 
 	const colorVal = value.currentColor ? value.currentColor : props.colorValue;
 
-	const globalIndicator = ( colorVal && colorVal.includes( 'var' ) ) ? 'uag-global-indicator' : '';
+	const pickIconColorBasedOnBgColorAdvanced = ( color ) => {
+		const result = /^#?([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})$/i.exec( color );
+		const parsed_color = result
```
Comment (RajkiranBagal) [inline src/components/color-control/advanced-pop-color-control.js:147]: JS variables should be camelCase

File: src/components/color-control/advanced-pop-color-control.js [hunk 2]
```diff
@@ -142,7 +142,26 @@ const AdvancedPopColorControl = ( props ) => {
...
+				b: parseInt( result[ 3 ], 16 ),
+		  }
+		: null;
+		if ( parsed_color ) {
+			const brightness = Math.round( ( ( parsed_color.r * 299 ) +
+						( parsed_color.g * 587 ) +
+						( parsed_color.b * 114 ) ) / 1000 );
+			const textColour = ( brightness > 125 ) ? 'black' : 'white';
+			return textColour;
+		}
```
Comment (RajkiranBagal) [inline src/components/color-control/advanced-pop-color-control.js:160]: What if parsedColor is null? Then nothing will return? There should be some default return either black or white.

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: src/components/color-control/advanced-pop-color-control.js (hunk 1 above)
Comment (AbhijitNage123) [reply src/components/color-control/advanced-pop-color-control.js]: Updated

File: src/components/color-control/advanced-pop-color-control.js (hunk 2 above)
Comment (AbhijitNage123) [reply src/components/color-control/advanced-pop-color-control.js:160]: Done

**Review by RajkiranBagal — APPROVED** [review]

**Review by supriyajjadhav — APPROVED** [review]

**Pull request merged**